demo/vmdefs/centos7-mezzanine/Vagrantfile

```diff
@@ -0,0 +1,29 @@
+VAGRANTFILE_API_VERSION = "2"
```

Comment:
Good point, I've added a `enabled` directory and symlinked the phpMyAdmin, Drools and Target VMs there (+ update to `{start,stop,destroy}_vms.sh`).